src/System Application/App/Cryptography Management/src/RSA.Codeunit.al

```diff
@@ -26,6 +26,16 @@ codeunit 1475 "RSA"
         RSAImpl.InitializeRSA(KeySize);
     end;
 
+    /// <summary>
+    /// Initializes a new instance of RSA with the specified key size and a pem private Key.
+    /// </summary>
+    /// <param name="KeySize">The size of the key in bits.</param>
+    /// <param name="PrivateKey">The private key as text.</param>
+    procedure InitializeRSA(KeySize: Integer; PrivateKey: Text)
+    begin
+        RSAImpl.InitializeRSA(KeySize, PrivateKey);
```

Comment:
Maybe you forgot to include the changes to the RSA Implementation file?

Reply:
Hi Jesper, I must have loaded the wrong branch/version into the PR. I'm sorry about that.
Since all functions require the XML structure, I thought it would be easier to provide a function that converts the private key into the XML structure.

Reply:
No worries. That has happened to all of us at some point 😉 Let me take this for another spin!